packages/generator-one-app-module/__tests__/app.js

```diff
@@ -119,7 +119,7 @@ describe('generator-one-app-module', () => {
           '@americanexpress/one-app-ducks': '^4.0.0',
           immutable: '^3.8.2',
           'prop-types': '^15.5.9',
-          'react-intl': '^3.6.0',
+          'react-intl': '^4.5.3',
```

Comment:
This is a major version update, did you test it on a `react-intl` enabled module? also worth a try on the `cultured-frankie` and see if the integration tests still pass

Reply:
I installed the latest version of `react-intl` to cultured frankie and run the integration tests and everything passes. Also verified this with a local module as well.

Reply:
So will we have an integration test PR in One App for this upgrade?

Reply:
at some point I think, a lot of the frankies need updating anyway